rpki-validator/src/main/java/net/ripe/rpki/validator3/rrdp/RrdpServiceImpl.java

```diff
@@ -225,6 +225,7 @@ private int processDownloadedSnapshot(RpkiRepository rpkiRepository, RpkiReposit
                     in,
                     (snapshotInfo) -> {
                         if (!notification.sessionId.equals(snapshotInfo.getSessionId())) {
+                            rrdpMetrics.update(notification.snapshotUri, ErrorCodes.RRDP_WRONG_SNAPSHOT_HASH);
```

Comment:
Shouldn't this be `ErrorCodes.RRDP_WRONG_SNAPSHOT_SESSION`? Or place this at line 185 where the hash comparison failed?

Reply:
It should be RRDP_WRONG_SNAPSHOT_SESSION, updated.